README.md

```diff
@@ -1,3 +1,25 @@
+# Example: Handling webhook from external service
+
+This example highlights:
+- Example endpoint, that should be a target of external service webhook. This endpoint will create GiftCard in Saleor
+- Example how to authorize webhook
```

Comment:
```suggestion
- Example how to authorize webhook using a secret header
- Add information about the target Saleor instance in multi-tenancy
```

Reply:
Adding the domain is important from the MT perspective, I think it's worth mentioning